Library/Formula/gambit-scheme.rb

```diff
@@ -32,7 +34,7 @@ def install
     # Don't enable this when using clang, per configure warning.
     args << "--enable-single-host" unless ENV.compiler == :clang
 
-    args << "--enable-shared" if build.include? "enable-shared"
+    args << "--enable-shared" if build.with? "shared"
 
     if ENV.compiler == :clang
```

Comment:
You removed slightly too much here :wink:. We need to keep the:

```ruby
args << "--enable-single-host" unless ENV.compiler == :clang
```
line & explanation, as obviously, we'll always be using something that isn't `:clang` now.

If you want to you can compress it into the `args = %W[` block.